docs/reference/connector/apis/list-connectors-api.asciidoc

```diff
@@ -0,0 +1,71 @@
+[role="xpack"]
+[[list-connector-api]]
+=== List connectors API
+
+preview::[]
+
+++++
+<titleabbrev>List connectors</titleabbrev>
+++++
+
+Returns information about all stored connectors.
+
+
+[[list-connector-api-request]]
+==== {api-request-title}
+
+`GET _connector`
+
+[[list-connector-api-prereq]]
+==== {api-prereq-title}
+
+* To sync data using connectors, it's essential to have the Elastic connectors service running.
+
+[[list-connector-api-path-params]]
+==== {api-path-parms-title}
+
+`size`::
+(Optional, integer) Maximum number of results to retrieve.
+
+`from`::
+(Optional, integer) The offset from the first result to fetch.
+
+[[list-connector-api-example]]
+==== {api-examples-title}
+
+The following example lists all connectors:
+
+////
+
+[source,console]
+--------------------------------------------------
+PUT _connector/connector-1
+{
+  "index_name": "search-google-drive",
+  "name": "Google Drive Connector",
+  "service_type": "google_drive"
+}
+
+PUT _connector/connector-2
+{
+  "index_name": "search-sharepoint-online",
+  "name": "Sharepoint Online Connector",
+  "service_type": "sharepoint_online"
+}
+
+--------------------------------------------------
+// TESTSETUP
```

Comment:
If we're not showing the response, we don't need this TESTSETUP. If you do want to keep it, we should have a TEARDOWN that deletes these two connectors.